Cargo.toml

```diff
@@ -29,7 +29,7 @@ protobuf = "2"
 rand = "0.8"
 regex = "1.7"
 reqwest = "0.11"
-ruma = {git = "https://github.com/nemored/ruma", branch = "http-1", features = ["api", "appservice-api-c", "client-api-c"] }
+ruma = {git = "https://github.com/nemored/ruma", branch = "http-1", features = ["api", "appservice-api", "client-api-c", "events"] }
```

Comment:
Can you check if we still need the `events` feature enabled?

Reply:
Tests are still passing with events removed